src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -94,7 +94,7 @@ List<String> hashWords = new ArrayList<String>();
         <form action="/users/<%=request.getSession().getAttribute("user") %>" method="POST">
           <div class="form-group">
             <label class="form-control-label">Edit Your About Me:</label>
-            <textarea rows="5" cols="75" name="About Me" placeholder="I'm currently a student at..."></textarea>
+            <textarea class="form-control"rows="5" cols="75" name="About Me" placeholder="I'm currently a student at..."></textarea>
```

Comment:
What does class=form-control actually do?

Reply:
it pretty much just gives the textarea or input area a 100% width and can style the area. In this case, it aligned the textarea box up with the "Sent Messages" and gave the textarea a more rounded and cooler look.